opm/material/fluidsystems/blackoilpvt/Co2GasPvt.cpp

```diff
@@ -50,17 +50,19 @@ Co2GasPvt(const std::vector<Scalar>& salinity,
     setActivityModelSalt(activityModel);
     setThermalMixingModel(thermalMixingModel);
 
+    co2Tables = Params();
```

Comment:
Will not this just do the entire table building over again, since it already would be default-initialized at this point?

Also in BrineCo2Pvt.cpp.

Reply:
Still applies.